### ISSUE TITLE
To what extend should we use the built in "recording" stuff for OSC

### ISSUE THREAD

**Issue by CharlesHolbrow** [fluid-music/fluid-music#5]
We have a decision to make. How should an `OscInputDeviceInstance` know when it is recording?

If we use the built in functionality:
- Some of the work is already done for us

If we make our own
- We have to redo some of the work in parallel

And how an `OscInputDevice` is created:
https://github.com/CharlesHolbrow/cybr/blob/0223cbb523f9d0efac850ae1f1f7b3c30119c644/Source/OscInputDevice.cpp#L46-L52

Notes about when and how the built in recording functions are used:
https://github.com/CharlesHolbrow/cybr/blob/0223cbb523f9d0efac850ae1f1f7b3c30119c644/Source/OscInputDeviceInstance.h#L30-L39

First Question: Where are the `CachedValues` that indicate "we are record enabled"  to the input devices currently. I believe when we call `transport.record(bool, bool)` from the message thread, all it does is update the ValueTree. The current methods are a handy way of communicating handling this communication'
